src/Microsoft.Data.Analysis/DataFrame.IO.cs

```diff
@@ -306,5 +309,112 @@ private static DataFrameColumn CreateColumn(Type kind, string[] columnNames, int
                 return ret;
             }
         }
+
+        /// <summary>
+        /// Writes a DataFrame into a CSV.
+        /// Follows pandas API.
```

Comment:
What does `Follows pandas API` mean?

Reply:
I'm not sure, but I found it in the summary of LoadCSV too, so I borrowed it for consistency.

Reply:
My feeling was this is similar to Pandas, but I didn't check myself.

Reply:
I don't think this line has value to callers.